packages/babel-plugin-named-asset-import/index.js

```diff
@@ -7,6 +7,64 @@ function namedAssetImportPlugin({ types: t }) {
 
   return {
     visitor: {
+      ExportNamedDeclaration(
```

Comment:
This code is so similar to the `ImportDeclaration`. It would be great if we could avoid duplicating it. That might be overly complicating things though.

Reply:
I extracted `generateNewSource` and `replaceNotVisitedSpecifiers` functions

Reply:
This is the implementation/code comparing result between `importDeclaration` and `exportDeclaration`. 
Please let me know if you think we can avoid more duplicate code

<img width="1210" alt="screen shot 2018-11-02 at 2 28 57 pm" src="https://user-images.githubusercontent.com/2850493/47888884-d179e280-deab-11e8-852d-3918126a09b0.png">